_source/logzio_collections/_metrics-sources/docker.md

```diff
@@ -50,26 +51,60 @@ shipping-tags:
     : Logz.io listener URL to ship the logs to.
       {% include log-shipping/replace-vars.html listener=true %}
 
+    LOGZIO_MODULES <span class="required-param"></span>
+    : Comma-separated list of Metricbeat modules to be enabled on this container
+      (formatted as `"module1,module2,module3"`).
+      To use a custom module configuration file, mount its folder to `/logzio/logzio_modules`.
+
     LOGZIO_TYPE <span class="default-param">`docker-collector-metrics`</span>
     : The log type you'll use with this Docker.
       This is shown under the `type` field in Kibana. \\
       Logz.io applies parsing based on `type`.
 
-    matchContainerName
+    LOGZIO_LOG_LEVEL <span class="default-param">`"INFO"`</span>
+    : The log level the module startup scripts will generate.
+
+    LOGZIO_ADDITIONAL_FIELDS
+    : Semicolon-separated list of additional fields to be included with each message sent
+      (formatted as `fieldName1=value1;fieldName2=value2`). \\
+      To use an environment variable as a value, format as `fieldName=$ENV_VAR_NAME`.
+      Environment variables must be the only value in the field.
+      Where an environment variable can't be resolved, the field is omitted.
+
+    Parameters for Docker Collector (Metrics)
+    {:.inline-header}
+
+    DOCKER_MATCH_CONTAINER_NAME
     : Comma-separated list of containers you want to collect the logs from.
       If a container's name partially matches a name on the list, that container's logs are shipped.
       Otherwise, its logs are ignored. \\
-      **Note**: Can't be used with skipContainerName
+      **Note**: Can't be used with `DOCKER_SKIP_CONTAINER_NAME`
 
-    skipContainerName
+    DOCKER_SKIP_CONTAINER_NAME
     : Comma-separated list of containers you want to ignore.
       If a container's name partially matches a name on the list, that container's logs are ignored.
       Otherwise, its logs are shipped. \\
-      **Note**: Can't be used with matchContainerName
+      **Note**: Can't be used with `DOCKER_MATCH_CONTAINER_NAME`
+
+    DOCKER_PERIOD <span class="default-param">`10s`</span>
+    : Sampling rate of metrics.
+      The Docker API takes up to 2 seconds to respond,
+      so we recommend setting this to `3s` or longer.
+
+    DOCKER_CERTIFICATE_AUTHORITY
+    : Filepath to certificate authority
+      for connecting to Docker over TLS.
+
+    DOCKER_CERTIFICATE
+    : Filepath to CA certificate
+      for connecting to Docker over TLS.
+
+    DOCKER_KEY
+    : Filepath to Docker key
+      for connecting to Docker over TLS.
 
-    <div class="info-box note">
-      By default, logs from docker-collector-logs and docker-collector-metrics containers are ignored.
-    </div>
+    By default, logs from docker-collector-logs and docker-collector-metrics containers are ignored.
```

Comment:
`By default, logs from docker-collector-logs and docker-collector-metrics containers are ignored.`
Doesn't make any sense. We collect metrics, not logs

Reply:
Since it's a stand-alone page, maybe we can take this comment out?

Reply:
What about this instead? Looks like it was just an oversight.

```
By default, metrics from docker-collector-logs and docker-collector-metrics containers are ignored.
```

Reply:
It's not true for the metric. I believe it's true for logs only :) let's take it out

Reply:
And will it report metrics on itself?

Reply:
I believe it does